pkg/virt-operator/resource/generate/components/validations_generated.go

```diff
@@ -8440,9 +8440,8 @@ var CRDsValidation map[string]string = map[string]string{
                 description: Hardware address of a Virtual Machine interface
                 type: string
               name:
-                description: 'Name of the interface, corresponds to name of the network
-                  assigned to the interface TODO: remove omitempty, when api breaking
-                  changes are allowed'
+                description: Name of the interface, corresponds to name of the network
+                  assigned to the interface
```

Comment:
I don't understand this part. AFAIU, the pod's interface name still has the `omitEmpty` property: https://github.com/kubevirt/api/blob/f1689bb1fb9d9be9ea42bbe0efc9b29fa94ee9f8/core/v1/types.go#L501

Reply:
I tried to explain it in the commit message. Perhaps you can help rephrase it to be clearer.
The TODO is wrong, it should be optional and not mandatory. One such scenario was presented there.